pyspeckit/spectrum/units.py

```diff
@@ -958,26 +667,38 @@ def _make_header(self, tolerance=1e-8):
             self.wcshead['CRPIX1'] = 1.0
             return True
 
-    def _update_from(self, obj):
-        """Copies some attributes of obj to self.
-        (this code copied partly from numpy.ma.core:
-        https://github.com/numpy/numpy/blob/master/numpy/ma/core.py)
+    @classmethod
+    def find_equivalencies(self, velocity_convention=None, refX=None, refX_units=None,
+                           center_frequency=None, equivalencies=[]):
         """
+        Utility function to add equivalencies from the velocity_convention 
+        and the center_frequency
+        """
+        # equivalency finding
+        if velocity_convention:
+            print("generating the equivalencies")
```

Comment:
need to remove print statements (or if they're really necessary, use `log.info`, where `log` is `astropy.log`)